scoresheets/Onlooker.tex

```diff
@@ -0,0 +1,18 @@
+
+The maximum time for this test is 5 minutes.
+
+\begin{scorelist}
+	\scoreheading{Main Goal}
+	\scoreitem[5]{500}{reports the activities performed by the main person}
```

Comment:
Missing *Deus Ex Machina reductions*

Reply:
There is no much need of DeM in this task. The robot is not supposed to interrupt people in the house. Asking the person to confirm if he/she is doing ActivityX would be meaningless. Besides, asking to repeat an activity is rude and unnatural. So the robot has to observe the people in the house silently.

A DeM can be used only if the robot fails to back to the operator (the start position) to report and asks the operator to come to the robot or wave or call the robot.